scss/jeet/_settings.scss

```diff
@@ -1,7 +1,5 @@
-/**
- * Grid settings.
- * All values are defaults and can therefore be easily overidden.
- */
+/// Grid settings.
+/// All values are defaults and can therefore be easily overidden.
 
```

Comment:
Let's say a column that's 1/2 is inside a column that's 1/3 width. Normally you'd just do `jeet-column(1/2 1/3)` with `$jeet-parent-first` set you'd do `jeet-column(1/3 1/2)`.